etc/DependencyInstaller.sh

```diff
@@ -456,15 +456,22 @@ _installDebianPackages() {
 _help() {
     cat <<EOF
 
-Usage: $0 -prefix=DIR
-       $0 -local
+Usage: $0                       Installs all of OpenROAD's dependencies no need to run -base or -common.
+                                                           Requires priviledged access. 
+       $0 -base                 Installs OpenROAD's dependencies using package managers 
+                                                           (-common must be executed in another command)
+       $0 -common               Installs OpenROAD's common dependencies (-base must be executed in another command)
+       $0 -prefix=DIR           Installs common dependencies in an existing user-specified directory. Only used with -common. This flag cannot be used with sudo or with root access.
+       $0 -local                Installs common dependencies in "$HOME/.local". Only used with -common. This flag cannot be used with sudo or with root access.
```

Comment:
```suggestion
Usage: $0
                                # Installs all of OpenROAD's dependencies no
                                #     need to run -base or -common. Requires
                                #     priviledged access.
                                #
       $0 -base
                                # Installs OpenROAD's dependencies using
                                #     package managers (-common must be
                                #     executed in another command).

       $0 -common
                                # Installs OpenROAD's common dependencies
                                #     (-base must be executed in another
                                #     command).

       $0 -prefix=DIR
                                # Installs common dependencies in an existing
                                #     user-specified directory. Only used
                                #     with -common. This flag cannot be used
                                #     with sudo or with root access.

       $0 -local
                                # Installs common dependencies in
                                #    "$HOME/.local". Only used with
                                #    -common. This flag cannot be used with
                                #    sudo or with root access.
```
Slightly formatted, this way we can just copy the output of the `-help` and put in the docs.